test/notifications/open_test.dart

```diff
@@ -76,6 +77,8 @@ void main() {
   final zulipLocalizations = GlobalLocalizations.zulipLocalizations;
 
   Future<void> init({bool addSelfAccount = true}) async {
+    debugDisableBetaCompleteDialog = true;
+    addTearDown(resetDebugDisableBetaCompleteDialog);
```

Comment:
nit: alternatively, could dispense with the separate "reset" method:
```suggestion
    debugDisableBetaCompleteDialog = true;
    addTearDown(() => debugDisableBetaCompleteDialog = false);
```